tests/begin_test.go

```diff
@@ -25,16 +28,129 @@ import (
 	"github.com/kylelemons/godebug/pretty"
 )
 
-func TestBegin(t *testing.T) {
-	// Call begin
+func TestBeginEmpty(t *testing.T) {
 	tmpdir, err := setUpTest()
 	if err != nil {
 		t.Fatalf("%v", err)
 	}
 	defer cleanUpTest(tmpdir)
 
+	expectedMan := schema.ImageManifest{
+		ACKind:    schema.ImageManifestKind,
+		ACVersion: schema.AppContainerVersion,
+		Name:      *types.MustACIdentifier("acbuild-unnamed"),
+		App: &types.App{
+			Exec:  nil,
+			User:  "0",
+			Group: "0",
+		},
+	}
+
+	checkMinimalContainer(t, path.Join(tmpdir, ".acbuild", "currentaci"), expectedMan)
+}
+
+func TestBeginLocalACI(t *testing.T) {
+	expectedMan := schema.ImageManifest{
+		ACKind:    schema.ImageManifestKind,
+		ACVersion: schema.AppContainerVersion,
+		Name:      *types.MustACIdentifier("acbuild-begin-test"),
+		Labels: types.Labels{
+			types.Label{
+				Name:  *types.MustACIdentifier("version"),
+				Value: "9001",
+			},
+		},
+		App: &types.App{
+			Exec:  types.Exec{"/bin/nethack4", "-D", "wizard"},
```

Comment:
[I certainly hope not](https://github.com/ComputerScienceHouse/bingehack4/pulls)